probe/endpoint/ebpf.go

```diff
@@ -99,7 +100,10 @@ var lastTimestampV4 uint64
 
 func tcpEventCbV4(e tracer.TcpV4) {
 	if lastTimestampV4 > e.Timestamp {
-		log.Errorf("ERROR: late event!\n")
+		// See https://github.com/weaveworks/scope/issues/2334
+		log.Debugf("tcp tracer received event with timestamp %v even though the last timestamp was %v. Stopping the eBPF tracker.", e.Timestamp, lastTimestampV4)
```

Comment:
Also, explaning the probable cause of this (the kernel bug) and indicating that upgrading the kernel will likely fix it would be helpful.